paths.js

```diff
@@ -2,10 +2,13 @@
  * Transforms `path` into its relative `base` version
  * If base isn't part of the path provided returns absolute path e.g. `~/app`
  */
-export const relativePath = (base = "", path = location.pathname) =>
-  !path.toLowerCase().indexOf(base.toLowerCase())
+export const relativePath = (base = "", path = "") => {
+  if (!path && typeof location != "undefined") path = location.pathname;
+
+  return !path.toLowerCase().indexOf(base.toLowerCase())
```

Comment:
I think we can safely drop the default value for the second argument here. This function is only called in two places: [here](https://github.com/molefrog/wouter/blob/8374709758d2f9abc2e2ea3be7869b2464ee67f5/static-location.js#L14C2-L14C2) and [here](https://github.com/molefrog/wouter/blob/main/use-location.js#L53). In both places, `path` is expected to be a string. 

Plus, I think it better to have this module not rely on anything related to location.

```suggestion
export const relativePath = (base = "", path) => 
  !path.toLowerCase().indexOf(base.toLowerCase())
```